src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -31,29 +32,17 @@ public interface IInvocationExpression : IExpression
         /// <summary>
         /// 'This' or 'Me' argument to be supplied to the method.
         /// </summary>
-        IExpression Instance { get; }
+        IOperation Instance { get; }
```

Comment:
Some people might consider `This` and `Me` as receiver instead of argument, the comment here would be confusing for them maybe?